python/sglang/srt/layers/moe/fused_moe_triton/fused_moe.py

```diff
@@ -638,6 +638,10 @@ def get_moe_configs(
     config_file_path = os.path.join(
         os.path.dirname(os.path.realpath(__file__)), "configs", json_file_name
     )
+    if int(os.getenv("SHARE_EXPERTS_FUSION_REPLICA", "0")) > 0:
```

Comment:
nit: wondering whether we can avoid creating a new subdirectory, because it seems the computation speed is determined by E (num experts) and may not be different when num experts come from fusion or non-fusion

Reply:
Ok, I agree.